devel/centos9_build_setup.yml

```diff
@@ -0,0 +1,76 @@
+---
+- hosts: cluster
+  become: yes
+  tasks:
+
+    - name: Enable CRB repo
+      command: yum config-manager --set-enabled crb
+
+    - name: Install required rpms
+      yum:
+        name:
+          - git
+          - gdb
+          - gcc
+          - python3
+          - perl
+          - automake
+          - autoconf
+          - libtool
+          - flex
+          - bison
+          - python3-pip
+          - openssl-devel
+          - libxml2-devel
+          - python3-devel
+          - libaio-devel
+          - libibverbs-devel
+          - librdmacm-devel
+          - readline-devel
+          - lvm2-devel
+          - glib2-devel
+          - userspace-rcu-devel
+          - libcmocka-devel
+          - libacl-devel
+          - sqlite-devel
+          - fuse-devel
+          - cups-devel
+          - dbus-devel
+          - docbook-style-xsl
+          - libarchive-devel
+          - libcap-devel
+          - libldb-devel
+          - libtalloc-devel
+          - libtdb-devel
+          - libtevent-devel
+          - libxslt
+          - openldap-devel
+          - pam-devel
+          - perl-ExtUtils-MakeMaker
+          - perl-Parse-Yapp
+          - perl-Test-Simple
+          - popt-devel
+          - python-tdb
+          - python-tevent
+          - gnutls-devel
+          - libtirpc
+          - libtirpc-devel
+          - libnsl2-devel
+          - rpcgen
+          - jansson-devel
```

Comment:
I haven't checked for a while the recent updates to above requirements. @spuiuk How did you come up with the above list of packages?

Reply:
A combination of a list Poornima gave me a while ago from her notes on building samba, from using rpmbuild to build samba and whatever other packages I had to install myself.  If I had to do this today, I would put this in a build container and use that instead.

Reply:
I remembered a few more details and added a bit more context to why I put this ansible playbook in the repo. I have put those details in the earlier comment.